breaking-changes.md

```diff
@@ -4,6 +4,14 @@ View [CHANGELOG.md](CHANGELOG.md) for more detail on releases. This file is only
 
 ## Version 2
 
+### Removed SourceFile#getLineNumberAtPos()
```

Comment:
Thanks, but could you move this to the bottom? It's not as important to know about as the other items.